Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -58,96 +58,33 @@ extension Store {
   }
 }
 
-extension ViewStore {
-  @available(
-    *, deprecated,
-    message:
-      "Dynamic member lookup is no longer supported for bindable state. Instead of dot-chaining on the view store, e.g. 'viewStore.$value', invoke the 'binding' method on view store with a key path to the value, e.g. 'viewStore.binding(\\.$value)'. For more on this change, see: https://github.com/pointfreeco/swift-composable-architecture/pull/810"
-  )
-  public subscript<Value>(
-    dynamicMember keyPath: WritableKeyPath<State, BindableState<Value>>
-  ) -> Binding<Value>
-  where Action: BindableAction, Action.State == State, Value: Equatable {
-    self.binding(
-      get: { $0[keyPath: keyPath].wrappedValue },
-      send: { .binding(.set(keyPath, $0)) }
-    )
-  }
-}
```

Comment:
Deprecations in this file are kept in order, so we can know when a particular interface was deprecated. Can you bring this back and wrap it with `#if compiler(>=5.4)` here?

Reply:
Just a clarification: I think this PR should be a diff of 2 lines that adds an `#if` above and `#endif` below this extension.

Reply:
Actually, I just did the same thing. 
- Added `#if compiler(>=5.4)`  and fixed the indentation. 
- Removed the deprecations in else part. 

However github diff visualizer makes it look complicated. On Xcode it's more clear

Reply:
The deprecations are in correct order at the moment

Reply:
Are you sure you're pushed? I pulled down your branch and am seeing the same as reflected on GitHub.

Reply:
Yes, I am quite confident this time. Here how it looks on my Xcode diff
<img width="1483" alt="Screen Shot 2021-09-18 at 02 11 42" src="https://user-images.githubusercontent.com/4898445/133862744-a4a5fe81-1cc6-47dd-96be-b536e6b2a12b.png">

 Although diff looks quite weird, line numbers are matching.